src/serve/index.ts

```diff
@@ -0,0 +1,48 @@
+import { EmulatorServer } from "../emulator/emulatorServer";
+
+const _ = require("lodash");
+
+const logger = require("../logger");
+
+const TARGETS: {
+  [key: string]:
+    | EmulatorServer
+    | { start: (o: any) => void; stop: (o: any) => void; connect: () => void };
+} = {
+  hosting: require("./hosting"),
+  functions: require("./functions"),
+  database: require("./database"),
+  firestore: require("./firestore"),
+};
+
+const _serve = async function(options: any) {
```

Comment:
And finally, add a return type, even though it's obvious: `...(options: any): Promise<void>`